### PR TITLE
wordsmithing, remove unused md file that has content on the rest of t…

### PR DESCRIPTION
**Pull request by CEKrause**
…he site

* [x] I have spellchecked my content using Microsoft Word, Grammarly, or otherwise.
* [ ] If required, update the [DEA Tech Alerts and Changelog](TechAlertsChangelog) and the [DEA Sandbox Updates banner](SandboxUpdatesBanner).

[TechAlertsChangelog]: https://github.com/GeoscienceAustralia/dea-knowledge-hub/blob/main/docs/tech-alerts-changelog/_tech_alerts_changelog.md
[SandboxUpdatesBanner]: https://bitbucket.org/geoscienceaustralia/datakube-apps/src/64c28bbf3d0e019d8940547a22f78b9bfd58d739/clusters/dea-sandbox/sandbox.yaml

### CODE REVIEW

**Comment by CEKrause** [comment]
Excellent. All the formatting worked as expected.

**Comment by benji-glitsos-ga** [comment]
@CEKrause  The "How to access data" article contains some content that I can't find anywhere else on the KH. It explains the meaning of: AWS S3, eCat, and DEA Maps.

Should we move those definitions to the glossary?

**Comment by CEKrause** [comment]
The eCat stuff isn't really needed - it's an internal GA term and we don't need to explain it to our users. The Maps content is on the DEA Maps page in the user guide. 
There was only a sentence that wasn't captured in the AWS S3 content, but I have put that back now onto the AWS page on the user guide.

**Comment by benji-glitsos-ga** [comment]
@CEKrause I've edited those category descriptions (you can read them to make sure you are ok with the wording) and also i've done minor edits to the paragraphs of content you added to those articles. I also used Grammarly to ensure that there are no grammar mistakes.

File: docs/data/theme/inland-water/_data.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 title: Inland water
-description: Browse inland water data products.
+description: Take a look at inland water bodies and assess their water extent, frequency, and variability.
```
Comment (robbibt) [inline docs/data/theme/inland-water/_data.yaml:2]: These products cover more than just water bodies (e.g. also event based and ephemeral processes like inland flooding)

```suggestion
description: Explore inland water across Australia, including changes in water extent, frequency, and variability over time.
```

File: docs/data/theme/sea-ocean-and-coast/_data.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 title: Sea, Ocean, and Coast
-description: Browse sea, ocean, and coast data products.
+description: Analyse the features and changes of coastal, tidal, and intertidal zones.
```
Comment (robbibt) [inline docs/data/theme/sea-ocean-and-coast/_data.yaml:2]:
```suggestion
description: Provide insights into Australia's coastal environments and their change through time
```

File: docs/data/theme/land-and-vegetation/_data.yaml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 title: Land and Vegetation
-description: Browse land and vegetation data products.
+description: See how the land surface and vegetation of an area have changed over time.
```
Comment (robbibt) [inline docs/data/theme/land-and-vegetation/_data.yaml:2]:
```suggestion
description: See how Australia's land surface and vegetation have changed over time.
```

File: docs/data/theme/baseline-satellite-data/_data.yaml [hunk 4]
```diff
@@ -1,5 +1,5 @@
 title: Baseline satellite data
-description: Browse baseline satellite data products.
+description: Browse through DEA's foundational data products which are used for a variety of purposes.
```
[the remark this answers is not in the document]
Comment (benji-glitsos-ga) [reply docs/data/theme/baseline-satellite-data/_data.yaml]: @robbibt @CEKrause I can add the word 'satellite' to it like this:

```
Browse through DEA's foundational data products, including satellite data and services that support DEA Surface Reflectance data production.
```

File: docs/data/theme/inland-water/_data.yaml (hunk 1 above)
Comment (benji-glitsos-ga) [reply docs/data/theme/inland-water/_data.yaml]: @CEKrause @robbibt 

> Analyse inland water across Australia, including changes in water extent, frequency, and variability over time.

File: docs/data/theme/sea-ocean-and-coast/_data.yaml (hunk 2 above)
Comment (robbibt) [reply docs/data/theme/sea-ocean-and-coast/_data.yaml]: @benji-glitsos-ga The current wording here isn't ideal - there's not really such thing as a tidal environment, and intertidal environments are a subset of coastal environment...

Comment (benji-glitsos-ga) [reply docs/data/theme/sea-ocean-and-coast/_data.yaml]: @robbibt My current wording was this:

> Explore Australia's dynamic coastal, tidal, and intertidal zones.

But based on your feedback, I could change it to:

> Explore Australia's dynamic coastal environments including tidal and intertidal zones.

Or:

> Explore Australia's dynamic coastal environments including intertidal zones.

Comment (robbibt) [reply docs/data/theme/sea-ocean-and-coast/_data.yaml]: How about this? "intertidal zone" should be singular, and this category technically includes marine/ocean data too in the high and low tide composites

> Explore Australia's marine and coastal environments including dynamic coastlines and the intertidal zone.

Although I'd be happy with something super simple too:

> Explore Australia's dynamic marine and coastal environments.

Comment (benji-glitsos-ga) [reply docs/data/theme/sea-ocean-and-coast/_data.yaml]: Nice - I'll go with the first one.

**Pull request merged**